app/views/questionnaire.py

```diff
@@ -457,6 +463,26 @@ def _get_front_end_navigation(answer_store, current_location, metadata):
         return None
 
 
+def get_page_title_for_block(schema_json, current_location):
```

Comment:
Not sure the benefit we will get by passing the block instead of the current_location?